lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max_by { |x| x.length }.to_i
```

Comment:
We may be dealing with strings here; enum could be `["a", "ab", "abc"]`. In that case, performing a `.to_i` on the longest string doesn't make sense.

Otherwise, this is reasonable.